RxCocoa/iOS/NSTextStorage+Rx.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NSTextStorage+Rx.swift
+//  Rx
+//
+//  Created by Segii Shulga on 12/30/15.
+//  Copyright © 2015 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+    
+    import Foundation
+    import UIKit
+#if !RX_NO_MODULE
+    import RxSwift
+#endif
+
+extension NSTextStorage {
+    
+    public var rx_delegate:DelegateProxy {
+        return proxyForObject(RxTextStorageDelegateProxy.self, self)
+    }
+    
+    public var rx_string:Observable<String> {
```

Comment:
This is snippet from CLLocationManager
```swift
/**
    Reactive wrapper for `delegate` message.
    */
    public var rx_rangingBeaconsDidFailForRegionWithError: Observable<(region: CLBeaconRegion, error: NSError)> {
        return rx_delegate.observe("locationManager:rangingBeaconsDidFailForRegion:withError:")
            .map { a in
                let region = try castOrThrow(CLBeaconRegion.self, a[1])
                let error = try castOrThrow(NSError.self, a[2])
                return (region: region, error: error)
            }
    }
```

so in this case this should probably be something like
```swift
public var rx_didProcessEditingRangeChangeInLength: Observable<(editedMask:NSTextStorageEditActions, editedRange:NSRange, delta:)> {
       ....
}
```